iam-policy-autopilot-lints/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "node_kind_literal"
+version = "0.1.0"
+authors = ["IAM Policy Autopilot Contributors"]
```

Comment:
I see that we don't have authors in the workspace's `Cargo.toml`, should we add it there and infer it? Not sure that works since we explicitly exclude this crate.